internal/ui/model.go

```diff
@@ -21,6 +22,8 @@ const (
 	compactListHeight = 10
 	prefixPadding     = 20
 	timeFormat        = "2006/01/02 15:04"
+	goosWindows       = "windows"
+	goosDarwin        = "darwin"
```

Comment:
You could use these constants in other files too

But maybe you could move the somewhere else